extensions/interactions/PencilCodeEditor/directives/pencil-code-editor-validation.service.spec.ts

```diff
@@ -86,7 +86,7 @@ describe('Pencil Code Editor Validation Service', () => {
       let rulesDict = rof.createNew('CodeEquals', inputBackend, {
         x: 'CodeString'
       });
-      let answergroup2 = agof.createNew([rulesDict], testOutcome2, [], null);
+      let answergroup2 = agof.createNew([rulesDict], testOutcome2, [], '');
```

Comment:
@mridul-netizen -- please note, this is **wrong**. The "tagged skill misconception ID" can be null, and in fact '' is not a valid value here. Please fix this ASAP.

/cc @vojtechjelinek

Reply:
@seanlip Made a fix PR #12778 .PTAL